db/migrate/20151210225310_create_test_client_requests.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTestClientRequests < ActiveRecord::Migration
```

Comment:
that feels a little edgy to me: http://stackoverflow.com/questions/12608928/running-migration-script-on-test-environment-only/12627244#comment43033589_12627244

Is the issue that you don't want a `test_client_requests` table sitting empty in production?